lib/internal/Magento/Framework/Controller/Test/Unit/Result/RedirectTest.php

```diff
@@ -67,16 +70,44 @@ public function testSetPath()
             $this->returnValue($params)
         );
         $this->assertInstanceOf(
-            \Magento\Framework\Controller\Result\Redirect::class,
+            Redirect::class,
             $this->redirect->setPath($path, $params)
         );
     }
 
-    public function testRender()
+    public function testRenderDataProvider()
```

Comment:
Starting data provider with `test` is not a good idea. Method name should describe data provided and not method under test,